### ISSUE TITLE
LayoutReader: why masking the tokens to predict?

### ISSUE THREAD

**Issue by SimeonZhang** [microsoft/unilm#495]
Why using the pseudo inputs to mask the tokens to predict instead of using the target tokens directly?
I think this training setting is not mentioned in the paper.

**Comment by zlwang-cs** [comment]
Thank you for your interests in LayoutReader. Token masking is a popular way in LM pre-training to improve the model robustness. Similar to the mask language model in BERT, we also follow UNILM's pre-training settings which includes pseudo inputs. If you have further questions about the training settings, please refer to the paper https://arxiv.org/pdf/1905.03197.pdf
